refnx/reflect/_mdsimulation.py

```diff
@@ -0,0 +1,307 @@
+from __future__ import print_function, division
+import numpy as np
+try:
+    from Bio.PDB import PDBParser
+except ImportError:
+    print('biopython must be installed to use MDSimulation')
+    pass
+try:
+    import periodictable as pt
+except ImportError:
+    print('periodictable must be installed to use automatically generate '
+          'scattering lengths, either install periodic table or use a .lgt '
+          'file.')
+    pass
+from refnx.analysis import Parameters
+from refnx.reflect import structure, Component
+
+
+class MDSimulation(Component):
+    """
+    Determines the scattering length density profile along the z-axis of a
+    molecular dynamics output file (PDB format, unless MDAnalysis is
+    installed). Lateral averaging occurs in the x-y plane.
+
+    Parameters
+    ----------
+    pdbfile: str
+        The path and name of the .pdb file or for which the reflectometry
+        should be found.
+    layer_thickness: float, optional
+        The thickness of the layers that the simulation cell should be sliced
+        into. This will depend on the size of the particles in the simulation,
+        for example a coarse-grained forcefield simulation will require thicker
+        layers to stop overbinning.
+    cut_off: float, optional
+        The thickness (in the z-dimension) of the volume of the bottom
+        simulation cell that should be ignored. This is to allow for the use of
+        a vacuum gap in the simulation cell. This thickness can be determined
+        by visualising the simulation or by running the box with a cut-off of 0
+        and investigating the SLD profile that is produced.
+    flip: bool, optional
+        False if the system should be read as is, True if the simulation cell
+        should be rotated through the xy-plane. This expects the neutrons or
+        X-ray to be incident at the bottom of the simulation cell (where
```

Comment:
So flip means the cell is inverted and what was at the bottom of the cell is not closest to the incident beam?